app/assets/javascripts/effective_pages/google_analytics.js

```diff
@@ -1,13 +1,11 @@
 // Add Google Analytics Code
 // Supports Turbolinks and Turbo
-document.addEventListener('turbolinks:load', function(event){
+document.addEventListener("turbolinks:load", function(event) {
   if(typeof(gtag) != 'function') { return }
 
-  var code = $('head').find('script[data-gtag-code]').first().data('gtag-code');
-  if(typeof(code) == undefined) { return }
-
-  gtag('config', code, {
-    'page_title': document.title,
-    'page_path': location.href.replace(location.origin, "")
+  gtag('event', 'page_view', {
+    page_title: event.target.title,
```

Comment:
not sure if we want to `event.target?.title` on these attributes or not

Reply:
we always have page titles, should be okay, this is good feedback ty